bin/cmp.js

```diff
@@ -1,8 +1,25 @@
 #!/usr/bin/env node
 
-require('yargs')
+import yargs from 'yargs/yargs';
+import * as auxCreate from "./commands/aux-create.js";
+import * as auxDrop from "./commands/aux-drop.js";
+import * as auxTest from "./commands/aux-test.js";
+import * as generate from "./commands/generate.js";
+import * as list from "./commands/list.js";
+import * as migrate from "./commands/migrate.js";
+import * as reset from "./commands/reset.js";
+import * as rollback from "./commands/rollback.js";
+
+yargs(process.argv.slice(2))
+    .scriptName("cmp")
     .usage('Contentful migration tooling.')
-    .commandDir('commands')
-    .recommendCommands()
```

Comment:
What does the recommendCommands() function do and why is it removed?

Reply:
It should be put back :) it gives suggestions to commands when a command is misspelled e.g. `cmp generata` -> `Did you mean generate?`